src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -1171,6 +1170,8 @@ private void DecodeInterceptsLocationAttributeExperimentalCompat(
             var attributeSyntax = arguments.AttributeSyntaxOpt;
             Debug.Assert(attributeSyntax is object);
             var attributeLocation = attributeSyntax.Location;
+            diagnostics.Add(ErrorCode.WRN_InterceptsLocationAttributeUnsupportedSignature, attributeLocation);
```

Comment:
We've added a warning, but it doesn't look like support was actually removed. Is that intended? Should we keep the tracking issue to remove the logic below at some point?

Reply:
That's correct, the warning is here to give folks notice that we will fully eliminate support here. At which point the warning will likely be upgraded to error.